src/components/taskDetails/TaskUpdateModal.tsx

```
@@ -29,6 +28,14 @@ function TaskUpdateModal({
     return (
         <Modal isOpen={isOpen} toggle={() => setIsOpen(false)}>
             <div className={styles.taskUpdateModal}>
+                <button
+                    className={styles.closeButton}
+                    onClick={() => setIsOpen(false)}
+                    data-testid="task-update-modal-close-button"
+                    aria-label="Close"
+                >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)